src/main/kube-auth-proxy/create-kube-auth-proxy.injectable.ts

```diff
@@ -4,18 +4,24 @@
  */
 import { getInjectable } from "@ogre-tools/injectable";
 import { KubeAuthProxy, KubeAuthProxyDependencies } from "./kube-auth-proxy";
+import { spawn } from "child_process";
 import type { Cluster } from "../../common/cluster/cluster";
 import path from "path";
 import { getBinaryName } from "../../common/vars";
 import directoryForBundledBinariesInjectable from "../../common/app-paths/directory-for-bundled-binaries/directory-for-bundled-binaries.injectable";
+import getKubeAuthProxyCertDirInjectable from "./kube-auth-proxy-cert.injectable";
+
+export type CreateKubeAuthProxy = (cluster: Cluster, environmentVariables: NodeJS.ProcessEnv) => KubeAuthProxy;
 
 const createKubeAuthProxyInjectable = getInjectable({
   id: "create-kube-auth-proxy",
 
-  instantiate: (di) => {
+  instantiate: (di): CreateKubeAuthProxy => {
     const binaryName = getBinaryName("lens-k8s-proxy");
     const dependencies: KubeAuthProxyDependencies = {
       proxyBinPath: path.join(di.inject(directoryForBundledBinariesInjectable), binaryName),
+      proxyCertPath: di.inject(getKubeAuthProxyCertDirInjectable),
+      spawn,
```

Comment:
This should be its own injectable, maybe within a folder `src/main/child-process` since we probably shouldn't have any of those calls within `renderer`